doc/modules/ROOT/pages/index.adoc

```diff
@@ -6,105 +6,280 @@
-done
+--
+
+Go::
++
+--
+[source, go]
+----
+import (
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+    config := hazelcast.NewConfig()
+	config.ClusterConfig.SetAddress("<EXTERNAL-IP>:5701")
+    config.ClusterConfig.SmartRouting = false
+	client, err := hazelcast.StartNewClientWithConfig(config)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
 ----
+--
 
-Check the services were created with by running the command:
+====
 
-[source, shell script]
+Finally, start the client application using the following command.
+
+[tabs]
+====
+
+Java::
++
+[source, bash]
+----
+cd java-unisocket
+mvn package 
+java -jar target/*jar-with-dependencies*.jar
+----
+--
+
+NodeJS::
++
+--
+[source, bash]
+----
+cd nodejs-unisocket
+npm install
+npm start
+----
+--
+
+Go::
++
+--
+[source, bash]
 ----
-$ kubectl get svc
-NAME          TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)          AGE
-hazelcast-0   LoadBalancer   10.175.248.67    35.195.77.97    5701:30915/TCP   11m
-hazelcast-1   LoadBalancer   10.175.254.105   104.155.70.30   5701:30440/TCP   11m
-hazelcast-2   LoadBalancer   10.175.253.232   34.78.182.50    5701:30305/TCP   11m
+cd go-unisocket
+go get github.com/hazelcast/hazelcast-go-client@v1.0.0-preview.4
+go run main.go
+----
+--
+
+====
+
+=== Clean Up
+
+To clean up the environment execute the following commands.
+
+[tabs]
+====
+
+Kubectl::
++
+
+--
+[source, bash]
+----
+kubectl delete pod/hz-hazelcast-0 pod/hz-hazelcast-1 pod/hz-hazelcast-2 service/hz-hazelcast
+kubectl delete -f https://raw.githubusercontent.com/hazelcast/hazelcast-kubernetes/master/rbac.yaml
+----
+--
+
+Helm::
++
+--
+[source, bash]
+----
+helm uninstall hz-hazelcast
+----
+====
+
+Unisocket is very simple to configure; however, it has one significant drawback - low performance. Using a load balancer is perfect for traffic distribution across stateless services; however, Hazelcast is not stateless. Actually, Hazelcast is more like a sharded database in which each member contains a different part of the data. That is why it’s suboptimal to randomly load balance the traffic. It effectively means that each operation needs to be internally migrated, because your data may be load balanced to member 1, while the partition for its key is stored in member 2. All in all, if performance is important for your use case, then you need to use smart client.
+
+== Smart Client
+
+Hazelcast smart client stores the mapping from key to member, which means that it sends the data directly to the member which contains the right data partition. This fact implies that in the Kubernetes environment, we need to provide access to each Hazelcast pod from the outside. A dynamic approach to this problem is to expose each pod with a separate service. Again, the service can be either `LoadBalancer` or `NodePort`. In a real-life scenario, that would probably be `NodePort`, since having a separate public IP for each pod is expensive.
```

Comment:
nit: the "client stores the mapping from key to member" phrase may be confusing for the reader
```suggestion
Hazelcast smart client is capable of mapping the given key with its owner member, which means that it sends the data directly to the member which contains the right data partition. This fact implies that in the Kubernetes environment, we need to provide access to each Hazelcast pod from the outside. A dynamic approach to this problem is to expose each pod with a separate service. Again, the service can be either `LoadBalancer` or `NodePort`. In a real-life scenario, that would probably be `NodePort`, since having a separate public IP for each pod is expensive.
```